tt_eager/tt_dnn/op_library/rotary_embedding/kernels/compute/rotary_embedding_llama.cpp

```diff
@@ -0,0 +1,119 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "compute_kernel_api/common.h"
+#include "compute_kernel_api/eltwise_binary.h"
+#include "compute_kernel_api/bcast.h"
+#include "compute_kernel_api/tilize.h"
+#include "compute_kernel_api/untilize.h"
+#include "compute_kernel_api/matmul.h"
+#include "debug/dprint.h"  // required in all kernels using DPRINT
```

Comment:
Remove

Reply:
I think we need matmul.h for the mm operations. Will remove the rest!